pxr/usd/usdMtlx/parser.cpp

```diff
@@ -239,6 +327,45 @@ ShaderBuilder::AddProperty(
         metadata[SdrPropertyMetadata->ImplementationName] = j->second;
     }
 
+    if (!isOutput) {
+        ParseMetadata(metadata, SdrPropertyMetadata->Label, element, "uiname");
+        ParseMetadata(metadata, SdrPropertyMetadata->Help, element, "doc");
+        ParseMetadata(metadata, SdrPropertyMetadata->Page, element, "uifolder");
```

Comment:
Any reason why these three literals, plus "enum" and "enumvalues" above aren't also added to the private tokens?

Reply:
Yes, but not necessarily a good one. These strings are only used as std::string on the MaterialX side of things. Would be best to have them as tokens as well since that prevents an unnecessary malloc/strcpy.